library-best-practices/JLBP-8.md

```diff
@@ -5,12 +5,12 @@ Libraries at 0.x versions and unstable features (which should be marked with
 annotations if they live in a stable library, as outlined in
 [JLBP-3](JLBP-3.md)) make no promises about stability. If they become widely
 used, they can cause problems when there are surface breakages (which is allowed
-per the rules of unstable functionality, explained in [JLBP-3](JLBP-3.md)). If
-such breakages can cause widespread pain, the unstable features can become
-"common-law GA." The preferable path is to promote such widely-used unstable
-libraries and features to stability, even if the functionality is not complete,
-because stability is not about completeness, it is about guarantees to not break
-users frequently or without warning.
+per the rules of unstable functionality, explained in [JLBP-3](JLBP-3.md)).
+To avoid widespread caused by such breakages, you can treat the unstable features as
```

Comment:
revert; this is not what was being said. I think just leave this section untouched in this PR though if it's confusing to you we do need to rewrite it.

Reply:
Reverted.

Created #374